src/mangata/mangata.js

```diff
@@ -1,166 +1,10 @@
-    console.log('automationFeeDetails: ', { executionFee: executionFee.toHuman(), xcmpFee: xcmpFee.toHuman() });
-
-    // Get a TaskId from Turing rpc
-    const taskId = await turingHelper.api.rpc.automationTime.generateTaskId(turingAddress, providedId);
-    console.log('TaskId:', taskId.toHuman());
-
-    // Send extrinsic
-    console.log('\n3. Sign and send scheduleXcmpTask call ...');
-    await turingHelper.sendXcmExtrinsic(xcmpCall, account.pair, taskId);
-
-    // Listen XCM events on Mangata side
-    console.log(`\n4. Keep Listening XCM events on ${parachainName} until ${moment(executionTime * 1000).format('YYYY-MM-DD HH:mm:ss')}(${executionTime}) to verify that the task(taskId: ${taskId}, providerId: ${providedId}) will be successfully executed ...`);
-    await listenEvents(mangataHelper.api, 'proxy', 'ProxyExecuted');
-
-    const isTaskExecuted = await listenEvents(mangataHelper.api, 'proxy', 'ProxyExecuted', executionTime);
-    if (!isTaskExecuted) {
-        console.log('Timeout! Task was not executed.');
-        return;
-    }
-
-    console.log('Task has been executed!');
-
-    console.log('\nWaiting 20 seconds before reading new chain states ...');
-    await delay(20000);
-
-    // Account’s reserved LP token after auto-compound
-    const newLiquidityBalance = await mangataHelper.mangata.getTokenBalance(liquidityTokenId, mangataAddress);
-    const newReservedBalanceBN = (new BN(newLiquidityBalance.reserved)).div(liquidityDecimalBN);
-    console.log(`\nAfter auto-compound, reserved ${poolName} is: ${newReservedBalanceBN} ${poolName} ...`);
-
-    const reservedPlanckDeltaBN = newLiquidityBalance.reserved.sub(liquidityBalance.reserved);
-    const reservedDeltaBN = reservedPlanckDeltaBN.div(liquidityDecimalBN);
-    console.log(`${account.name} has compounded ${reservedDeltaBN.toString()} more ${poolName} ...`);
-
-    console.log('\n5. Cancel task ...');
-    const cancelTaskExtrinsic = turingHelper.api.tx.automationTime.cancelTask(taskId);
-    await sendExtrinsic(turingHelper.api, cancelTaskExtrinsic, keyPair);
-
-    const nextExecutionTime = executionTime + TASK_FREQUENCY;
-    const nextExecutionTimeout = calculateTimeout(nextExecutionTime);
-
-    console.log(`\n6. Keep Listening events on ${parachainName} until ${moment(nextExecutionTime * 1000).format('YYYY-MM-DD HH:mm:ss')}(${nextExecutionTime}) to verify that the task was successfully canceled ...`);
-
-    const isTaskExecutedAgain = await listenEvents(mangataHelper.api, 'proxy', 'ProxyExecuted', nextExecutionTimeout);
-    if (isTaskExecutedAgain) {
-        console.log('Task cancellation failed! It executes again.');
-        return;
-    }
-    console.log("Task canceled successfully! It didn't execute again.");
+    const mangataDevAutoCompound = new MangataDevAutoCompound();
+    await mangataDevAutoCompound.initialize();
+    await mangataDevAutoCompound.run();
```

Comment:
If there are only three lines in this file, what is the point of extraction?

Please get rid of the new file MangataDevAutoCompound and put all logic back to src/mangata.js.